spec/support/feature_flag_helper.rb

```diff
@@ -1,9 +1,27 @@
 module FeatureFlagHelper
```

Comment:
I think we can simplify a bit?

```ruby
module FeatureFlagHelper
  def activate_feature(name)
    initialize_stub
    allow(Settings.feature_flags).to receive(:send).with(name).and_return true
  end

  def deactivate_feature(name)
    initialize_stub
    allow(Settings.feature_flags).to receive(:send).with(name).and_return false
  end

  private

  def initialize_stub
    @_stub_initialized ||= allow(Settings.feature_flags).to receive(:send).and_call_original
  end
end
```

Reply:
If this gets around the issue I mentioned in the [commit message](https://github.com/DFE-Digital/find-teacher-training/pull/807/commits/b884e3e56e0bdca0ce5600db0342036ff1937e41), then sure!

> Needed to change the feature flag helper so that it could handle tests
> that tested code that doesn't check flags other than those that the test
> cares about. Without this change, when non-stubbed features are checked
> for activation, rspec would raise an error.

I'll try it out when I have a moment, or if you have the time feel free too!

Reply:
I tried it out — it does the job of making the suite green!

Reply:
Thanks. Changed.